src/styles/Films.module.css

```diff
@@ -0,0 +1,84 @@
+@media screen and (min-width: 320px) {
+  .pictureCard {
+    padding: 1rem;
+    margin: 0 1rem;
+    width: 90%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .pictureCard:nth-child(odd) {
+    background-color: #194569;
+    color: white;
+  }
+
+  .pictureCard:nth-child(even) {
+    background-color: #5f84a2;
+    color: white;
+  }
+
+  .title {
+    margin-bottom: 0.25rem;
+    font-size: 1.5rem;
+  }
+
+  .image {
+    width: 100%;
+    max-height: 250px;
+    object-fit: cover;
+    border: 1px black solid;
+  }
+
+  .detailBtn {
+    background: transparent;
+    width: fit-content;
+    padding: 0.5rem 1rem;
+    color: white;
+    border: 1px solid white;
+    margin: 0.5rem 0;
+  }
+
+  .detailBtn:hover {
+    cursor: pointer;
+    transform: translateY(-1px);
+    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.2), 0 1px 2px rgba(0, 0, 0, 0.4);
+  }
+
+  .detailBtn:active {
+    transform: translateY(1px);
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .picturesList {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+  }
+
+  .pictureCard {
+    width: 31%;
+    margin: 0.5rem;
+    box-shadow: 8px 4px 16px rgba(0, 0, 0, 0.4);
+  }
+
+  .title {
+    margin-bottom: 0.5rem;
+    font-size: 2rem;
+  }
+}
+
+@media screen and (min-width: 1280px) {
+  .pictureCard {
+    width: 20%;
+    margin: 1rem;
+  }
+}
+
+@media screen and (min-width: 1600px) {
+  .pictureCard {
+    width: 18%;
+  }
+}
```

Comment:
- Hey great design you have, however, we need to follow the design layout and also use either one of the used background colors, therefore, maintaining it throughout the design. Currently, you've not followed the design layout for each card, as the parent container is a two-column card having its children (each card) in rows taking 50% of the parent width, giving them the flexibility to span across each other. Please, see the design below and try make some adjustments to your styles to improve your design.


*your design*
![Screenshot from 2022-12-01 01-07-07](https://user-images.githubusercontent.com/59397159/204935528-5d6d679e-27d5-4ec0-95f1-f043bb24550c.png)

*design template*
![Screenshot from 2022-11-27 06-04-09](https://user-images.githubusercontent.com/59397159/204935536-a12eb65b-fd79-45de-a1aa-e73fc6d7600d.png)

Reply:
@zieeco can you show me how to get the exact colors..? Because I am using the blue version that is in the same design template, but I don't know how to match the exact colors... Also, 2 columns with these pictures does not make sense..

Reply:
Styling is changed to fit the template as much as possible, but I still used the blue color palette